matrix/plugins/glitch/utils.py

```diff
@@ -0,0 +1,17 @@
+from juju.client.client import ClientFacade
+import subprocess
+import yaml
+
+async def is_leader(model, rule, unit_name):
```

Comment:
johnsca: it's a kluge, so it felt like it belonged here rather than in the library that we're offering as the "correct" way of doing things.

I can move it into libjuju if you'd rather, though.